tests/test_base.py

```diff
@@ -3,7 +3,11 @@
 import unittest.mock
 
 
-from tests.base import LoggingTestCase, _CaptureLogHandler
+from tests.base import LoggingTestsMixin, _CaptureLogHandler
+
+
+class LoggingTestCase(LoggingTestsMixin):
+    pass
```

Comment:
Why?

Reply:
I've changed this into a more realistic use case with a class both implementing this mixin as well as a proper TestCase class. That's how it's going to be used when it's implemented as well.

Reply:
OK now I realise what was going on here. The mixin relies on methods from `TestCase` but it doesn't actually subclass it. Hence the class above is needed so it can subclass `TestCase`.